src/ucp/Makefile.am

```diff
@@ -161,6 +163,4 @@ libucp_la_SOURCES = \
 	wireup/wireup.c \
 	wireup/wireup_cm.c \
 	stream/stream_send.c \
-	stream/stream_recv.c \
-	am/eager_single.c \
```

Comment:
if it's moved to keep alphabetical order, maybe it's worth moving `stream*` as well?